City-of-Helsinki/drupal-helfi-sote
# UHF-7782: Hide sotebot on health station pages

# [UHF-7782](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7782)

## What was done
- Hide sotebot block visibility on health station pages

## How to install

1. set up SOTE instance (try to get the production database if possible as it is easier to verify the block visibility in required paths)
2. checkout this branch: `git checkout UHF-7782_sote_chatbot_config`
3. import config and clear cache: `make drush-cim drush-cr`
4. set the config values for the "IBM Chat App" block from the comment https://helsinkisolutionoffice.atlassian.net/browse/UHF-7766?focusedCommentId=75469


## How to test
From original PR:
1. view the paths defined in the ticket and verify that the chatbot is visible
2. view (at least some) paths outside of that scope and verify that the chatbot is not visible

New: 
1. Check that new sotebot is not visiblle on health station pages (/terveydenhoito/terveysasemat, /halsovard/halsostationer, /health-care/health-stations) or their subpages
2. Check the block is not visible on dental service pages (/terveydenhoito/hammashoito, /halsovard/tandvard, /health-care/dental-care) or their subpages
